runtime-sdk/src/modules/consensus/mod.rs

```diff
@@ -231,6 +255,27 @@ impl Module {
         // Set genesis parameters.
         Self::set_params(genesis.parameters);
     }
+
+    #[handler(call = "consensus.RoundRoot", internal)]
+    fn internal_round_root<C: TxContext>(
```

Comment:
Why do we return only one root, if we already fetched both of them?

Reply:
This (internal) call version is intended to be used from smart contract modules (via the subcalls feature). It was suggested to keep the response as simple as possible to make the cbor parsing in contracts (solidity) easier. 

Higher level code (e.g. a runtime module) would use the `round_roots` method directly and it would get the more structured result with both hashes.

Reply:
> It was suggested to keep the response as simple as possible to make the cbor parsing in contracts (solidity) simpler.

Makes sense.

> This (internal)-call version is intended to be used from smart contract modules (via the subcalls feature). 

Yes, and the caller will now have to pay double gas for the same operation. That's why we could return both roots.

Reply:
What is the use case where the caller would actually need both roots for the same round? Usually you would fetch the roots to verify a specific Merkle proof.